src/framework/ui/card/card.component.tsx

```diff
@@ -0,0 +1,230 @@
+        paddingVertical: headerStyles.headerPaddingVertical,
+        paddingHorizontal: headerStyles.headerPaddingHorizontal,
+      },
+      body: {
+        paddingVertical: bodyStyles.bodyPaddingVertical,
+        paddingHorizontal: bodyStyles.bodyPaddingHorizontal,
+      },
+      footer: {
+        paddingVertical: footerStyles.footerPaddingVertical,
+        paddingHorizontal: footerStyles.footerPaddingHorizontal,
+      },
+      title: {
+        fontFamily: titleStyles.titleFontFamily,
+        fontSize: titleStyles.titleFontSize,
+        fontWeight: titleStyles.titleFontWeight,
+        lineHeight: titleStyles.titleLineHeight,
+        color: titleStyles.titleColor,
+        marginHorizontal: titleStyles.titleMarginHorizontal,
+      },
+      description: {
+        fontFamily: descriptionStyles.titleFontFamily,
+        fontSize: descriptionStyles.titleFontSize,
+        fontWeight: descriptionStyles.titleFontWeight,
+        lineHeight: descriptionStyles.titleLineHeight,
+        color: descriptionStyles.descriptionColor,
+        marginHorizontal: descriptionStyles.descriptionMarginHorizontal,
+      },
+      accent: {
+        backgroundColor: accentStyles.accentBackgroundColor,
+        height: accentStyles.accentHeight,
+      },
+    };
+  };
+
+  private renderDivider = (): DividerElement => {
+    return (
+      <Divider/>
+    );
+  };
+
+  private renderHeader = (headerStyles: HeaderStyles): CardHeaderElement => {
+    const header: CardHeaderElement = this.props.header();
+
+    return React.cloneElement(header, {
+      headerStyle: [styles.header, headerStyles.style, header.props.style],
+      accentStyle: headerStyles.accent,
+      titleStyle: headerStyles.title,
+      descriptionStyle: headerStyles.description,
+    });
+  };
+
+  private renderFooter = (style: StyleType): CardFooterElement => {
+    const footer: CardFooterElement = this.props.footer();
+
+    return React.cloneElement(footer, {
+      style: [styles.footer, footer.props.style, style],
+    });
+  };
+
+  private renderBody = (style: StyleType): React.ReactNode => {
+    return (
+      <View style={[styles.body, style]}>
+        {this.props.children}
+      </View>
+    );
+  };
+
+  private renderComponentChildren = (style: StyleType): React.ReactNodeArray => {
+    const { header, footer } = this.props;
+    const headerStyles: HeaderStyles = {
+      style: style.header,
+      accent: style.accent,
+      title: style.title,
+      description: style.description,
+    };
+
+    return [
+      header && this.renderHeader(headerStyles),
+      this.renderBody(style.body),
+      footer && this.renderFooter(style.footer),
+    ];
+  };
+
+  public render(): CardElement {
+    const { themedStyle, style, children, ...restProps } = this.props;
+    const { container, ...childrenStyles } = this.getComponentStyle(themedStyle);
+    const [header, body, footer] = this.renderComponentChildren(childrenStyles);
+
+    return (
+      <View
```

Comment:
Would it be better to make it `TouchableOpacity` with `activeOpacity={1.0}` by default? As a user, I expect it to be clickable